SalesforceSDK/Core/Auth/OAuth2.cs

```diff
@@ -323,30 +298,53 @@ public static string ComputeFrontDoorUrl(string instanceUrl, string displayType,
         /// <returns>Boolean based on if the refresh auth token succeeded or not</returns>
         public static async Task<Account> RefreshAuthToken(Account account)
```

Comment:
sure - I didn't touch the names cause I didn't want to change the public surface but if you think these methods are only used internally I could change it and the others in the file